plugins/mac-syphon/syphon.m

```diff
@@ -1,13 +1,19 @@
-#import <Cocoa/Cocoa.h>
+#define GL_SILENCE_DEPRECATION
```

Comment:
Note to self, as a conversation to avoid merging: This should be removed again

Reply:
Correct. :D

Reply:
Fixed.